Applications/VpView/CMakeLists.txt

```diff
@@ -158,73 +160,7 @@ set(vpViewSources
   vtkVpTrackModel.cxx
 )
 
-set(vpViewUI
-  qtInformaticsDialog.ui
-  qtNormalcyDialog.ui
-  qtRenderWindowDialog.ui
-  vpConfigure.ui
-  vpCreateEventDialog.ui
-  vpEditTrackTypesDialog.ui
-  vpExternalProcessDialog.ui
-  vpGraphModelWidget.ui
-  vpMergeTracksDialog.ui
-  vpObjectInfoPanel.ui
-  vpObjectSelectionPanel.ui
-  vpProjectEditor.ui
-  vpQtViewer3dDialog.ui
-  vpQtViewer3dWidget.ui
-  vpSelectTimeIntervalDialog.ui
-  vpTimelineDialog.ui
-  vpTrackAttributeColorDialog.ui
-  vpTrackAttributesPanel.ui
-  vpTrackColorDialog.ui
-  vpView.ui
-)
-
-if(VISGUI_ENABLE_SUPER3D)
-  list(APPEND vpViewUI
-    vpSuperResDepthViewer.ui
-    vpSuperResViewer.ui
-    vpSuperResWidget.ui
-    )
-endif()
-
-set(vpViewHeaders
-  vpApplication.h
-  vpConfigureDialog.h
-  vpCreateEventDialog.h
-  vpEditTrackTypesDialog.h
-  vpEntityConfigWidget.h
-  vpExternalProcessDialog.h
-  vpFileDataSource.h
-  vpGraphModelHelper.h
-  vpGraphModelView.h
-  vpGraphModelWidget.h
-  vpFrameMap.h
-  vpInformaticsDialog.h
-  vpMergeTracksDialog.h
-  vpObjectInfoPanel.h
-  vpObjectSelectionPanel.h
-  vpProject.h
-  vpProjectEditor.h
-  vpProjectList.h
-  vpQtViewer3d.h
-  vpQtViewer3dDialog.h
-  vpQtViewer3dWidget.h
-  vpSelectTimeIntervalDialog.h
-  vpSessionView.h
-  vpTimelineDialog.h
-  vpTabWidget.h
-  vpTrackAttributesPanel.h
-  vpTrackColorDialog.h
-  vpTreeView.h
-  vpVideoAnimation.h
-  vpView.h
-  vpViewCore.h
-  vtkVpTrackModel.h
-)
-
-set(vpViewResources ../../Icons/vpView.qrc)
+set(vpViewResSources ../../Icons/vpView.qrc)
```

Comment:
Is this a typo?

Reply:
Nope; "ResSources" → "Res[ource] Sources". See also "AmSources" (and we used to have a ton of "UiSources", "MocHeaders", etc. that AUTOMOC/AUTOUIC allowed us to eliminate).

Reply:
Regarding the large set of warnings - with current version, most is still the same. The first part of the warnign is now gone, so now starts with 

> CMake Warning (dev) in Libraries/VgGuiFramework/CMakeLists.txt:
>   Policy CMP0071 is not set: Let AUTOMOC and AUTOUIC process GENERATED files.
>   Run "cmake --help-policy CMP0071" for policy details.  Use the cmake_policy
>   command to set the policy and suppress this warning.
> 
>   For compatibility, CMake is excluding the GENERATED source file(s):
> 
>     "C:/Vision/diva/vivia/build-qt5/Libraries/VgGuiFramework/am_contextMenu.h"
>     "C:/Vision/diva/vivia/build-qt5/Libraries/VgGuiFramework/ActionManagerDialog.h"
> .
> .
> .